ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -182,12 +182,26 @@ PROXY_SWITCH=""
 if [[ "$ENABLE_PROXY" == "true" ]]; then
   # Get the proxy information from the previous steps, and replace the value here
   proxy_private_url=$(< "${SHARED_DIR}/proxy_private_url")
-  TRUST_BUNDLE_FILE="bundle_file"
-  if [[ -z "${proxy_private_url}" ]] || [[ -z "${TRUST_BUNDLE_FILE}" ]]; then
-    echo -e "The http_proxy, the http_proxy and the additional_trust_bundle_file are mandatory."
+  TRUST_BUNDLE_FILE="${SHARED_DIR}/bundle_file"
+  if [[ -z "${proxy_private_url}" ]]; then
+    echo -e "The http_proxy, and https_proxy URLs are mandatory when specifying use of proxy."
     exit 1
   fi
-  PROXY_SWITCH="--http-proxy ${proxy_private_url} --https-proxy ${proxy_private_url} --additional-trust-bundle-file ${TRUST_BUNDLE_FILE}"
+  if [[ -f "${TRUST_BUNDLE_FILE}" ]]; then
+    echo -e "Using proxy with requested additional trust bundle"
+    PROXY_SWITCH="--http-proxy ${proxy_private_url} --https-proxy ${proxy_private_url} --additional-trust-bundle-file ${TRUST_BUNDLE_FILE}"
+  else
+    echo -e "Using proxy with no additional trust bundle provided"
+    PROXY_SWITCH="--http-proxy ${proxy_private_url} --https-proxy ${proxy_private_url}"
```

Comment:
Let us set it as the default value. If ${TRUST_BUNDLE_FILE}, we replace it with the new one.

Reply:
done